manifests/4.6/cluster-logging.v4.6.0.clusterserviceversion.yaml

```diff
@@ -310,9 +319,7 @@ spec:
                 - cluster-logging-operator
                 env:
                   - name: WATCH_NAMESPACE
-                    valueFrom:
-                      fieldRef:
-                        fieldPath: metadata.annotations['olm.targetNamespaces']
+                    value: ""
```

Comment:
Although this is the recommendation from the operator-sdk docs, I believe it now allows to watch also for `ClusterLogging` resources in all namespaces. WDYT?

IMHO we can fix the limitation in the `clusterlogging_controller` by extending the watch predicate to watch for create/update/delete/generic events where the event object is in `openshift-logging`. (See https://github.com/openshift/cluster-logging-operator/blob/master/pkg/controller/clusterlogging/clusterlogging_controller.go#L41)

Reply:
I'm not sure we need to change this... `ElasticsearchOperator` also watches all namespaces but follows this pattern as well:
https://github.com/openshift/elasticsearch-operator/blob/master/manifests/4.6/elasticsearch-operator.v4.6.0.clusterserviceversion.yaml#L269-L272

Reply:
@ewolinetz I am not sure if this compatible with the updated operator-sdk 0.18 and the underlying controller-runtime 0.6.x

Reply:
I reverted this - looks like it not necessary (for now, in future if we implement per-namespace forwarders we do that) Need another LGTM please.